.github/workflows/build-vm.yaml

```diff
@@ -197,14 +186,17 @@ jobs:
             --protected-settings "{\"commandToExecute\":\"powershell Get-AksEdgeKubeConfig -KubeConfigPath C:\\Scripts -Confirm:\$false; \
                 kubectl get pods -A -v6 --kubeconfig C:\\scripts\\config; \
                 az login --identity; \
-                az extension add --name connectedk8s; \
-                az extension add --name azure-iot-ops; \
+                curl -L -o connectedk8s-1.10.0-py2.py3-none-any.whl https://github.com/AzureArcForKubernetes/azure-cli-extensions/raw/refs/heads/connectedk8s/public/cli-extensions/connectedk8s-1.10.0-py2.py3-none-any.whl; \
```

Comment:
nit: this Preview version of `connectedk8s` is only required when you want to use OIDC issuer stuff with Secret Sync and Key vault. Probably not needed since there is no usage of this feature in the repo.

Reply:
ah yes, thx